GOVERNANCE.md

```diff
@@ -244,7 +277,153 @@ The current NumFOCUS Subcommittee consists of:
+be nominated at any time again in the future, three months after the previous
+nomination at the earliest.
+
+### Current Core Contributors
+
+<!-- we could list them here or link to the pymc-devs organization, however,
+linking to the pymc-devs org would require the core team not being secret.
+I actually don't know if this is already the case, if it's possible even nor
+obviously how it would be done. I do believe it's important to be clear and
+explicit about who are core contributors, especially for people interested
+in joining the project to reach out, but also for organizations and meet-ups
+to ask for speakers.
+
+Regarding the reaching out, we do have the project mail and some people
+may not desire being contacted privately. I also think we should have some
+rules or guidance on this here and/or in the community/contributing pages.-->
+
+## Recurring Contributors
+Recurring Contributors are those individuals who can participate from
+The Project private communication channels.
+
+### Recurring Contributor membership
+Recurring Contributors can nominate any Contributor to participate in the
+Project private communication channels and become a Recurring Contributor.
+For the nomination to go forward, it has to be ratified by
+the Steering Council and the BDFL.
+For a nomination to be rejected, clear reasoning behind the decision must be
+shared with the rest of the team. People whose nomination has been rejected can
+be nominated at any time again in the future, three months after the previous
+nomination at the earliest.
+
+#### Current Recurring Contributors
+
+- everyone on slack! This may be too long so it could be a good idea to
+format as a table instead or to put within a `<details>` tag so
+it's hidden by default while still available publicly
+
+## Conflict of interest
+
+It is expected that the BDFL, Council Members and Contributors will be
+employed at a wide range of companies, universities and non-profit organizations.
+Because of this, it is possible that Members will have conflict of interests.
+Such conflict of interests include, but are not limited to:
+
+- Financial interests, such as investments, employment or contracting work,
+  outside of The Project that may influence their work on The Project.
+- Access to proprietary information of their employer that could potentially
+  leak into their work with the Project.
+
+All members of the Council, BDFL included, shall disclose to the rest of the
+Council any conflict of interest they may have. Members with a conflict of
+interest in a particular issue may participate in Council discussions on that
+issue, but must recuse themselves from voting on the issue. If the BDFL has
+recused themselves for a particular decision, they will appoint a substitute
+BDFL for that decision.
+
+## Team Organization
+<!-- Open questions
+
+1. do we want to give commit rights to core contributors to some repositories
+only based on the team they are part of?
```

Comment:
Yes, IMO there should be no "core-devs" GitHub team, but just {pymc3,examples}-{Triage,Write,Admin}.
Possibly even leaving the Triage level out and replacing it with proper branch protection (review required before merging), because Triage doesn't give permission to cancel/re-run CI jobs, but that's something we often need..

Plus a steering council Team that has Admin level on everything

Reply:
That sounds like a sensible mapping of our structure to github. I wanted to keep everything github agnostic to begin with, once the tiers look decided we can add that to the doc